components/pages/searchPageLayout/SearchPageLayout.tsx

```diff
@@ -70,43 +67,56 @@ const SearchPageLayout = ({ searchQuery }: { searchQuery?: string }) => {
           [mapFacetsToFilters[facetDefinition]]: [...values],
         }
       }
-
       return acc
     },
     {} as { [key: string]: keyof SearchFilters[] }
   )
 
-  const { data, isLoading } = useSearchWithPaginationQuery({
-    q: { all: q },
-    offset: 0,
-    limit: 10,
-    filters: {
-      branchId: branchIds,
-      ...facetsForSearchRequest,
+  const { data, isLoading } = useSearchWithPaginationQuery(
+    {
+      q: { all: q },
+      offset: 0,
+      // TODO: This should be configurable.
+      limit: 10,
+      filters: {
+        branchId: branchIds,
+        ...facetsForSearchRequest,
+      },
     },
-  })
+    {
+      enabled: q?.length > 0,
+    }
+  )
 
-  const { data: dataFacets, isLoading: isLoadingFacets } = useSearchFacetsQuery({
-    q: { all: q },
-    facetLimit: 100,
-    facets: facetDefinitions,
-    filters: {
-      branchId: branchIds,
-      ...facetsForSearchRequest,
+  const { data: dataFacets, isLoading: isLoadingFacets } = useSearchFacetsQuery(
+    {
+      q: { all: q },
+      facetLimit: 100,
+      facets: facetDefinitions,
+      filters: {
+        branchId: branchIds,
+        ...facetsForSearchRequest,
+      },
     },
-  })
+    {
+      enabled: q?.length > 0,
```

Comment:
I'd say because the target audience is children they might not be able to easily understand that they need min 3 letters. It was the project specs and Tue pushing for it to be 3 chars on the adult page. But here there is no project specs that micro manage us in the same way, and we can totally make queries with just 1 letter.